lib/src/async_queue.dart

```diff
@@ -59,12 +59,13 @@ class AsyncQueue<T> {
   /// the process was cancelled.
   Future _processNextItem() {
     var item = _items.removeFirst();
-    return _processor(item).then((_) {
+    return _processor(item).then((_) async {
       if (_items.isNotEmpty) return _processNextItem();
```

Comment:
If the method is `async`, this should be `return await`. Otherwise it's easy to accidentally refactor without realizing this is asynchronous.

Reply:
done.